Tests/packages.config

```diff
@@ -10,6 +10,7 @@
   <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net452" />
   <package id="NodaTime" version="1.3.4" targetFramework="net452" />
   <package id="NUnit" version="2.6.4" targetFramework="net452" />
+  <package id="NUnitTestAdapter" version="2.1.1" targetFramework="net452" />
```

Comment:
Was this required? If so why? By looking at the `.csproj` it doesn't look like it's required for compilation, but maybe it's required for running the tests within VS?

Reply:
To continue commenting: No, it's not required for compilation. I wanted to remove this dependency when I'd be finished anyway, and yes I run unit tests within VS this way. Is there any other way?